README.md

```diff
@@ -37,14 +37,8 @@ Development
   * Developers should [configure git to sign all tags](http://micropipes.com/blog//2016/08/31/signing-your-commits-on-github-with-a-gpg-key/) and upload their PGP fingerprint to https://login.mozilla.com
   * The signature verification will eventually become a requirement to shipping a release to staging & prod: the tag being deployed in the pipeline must have a matching tag in git signed by a project owner. This control is designed to reduce the risk of a 3rd party GitHub integration from compromising our source code.
 * [ ] enable security scanning of 3rd-party libraries and dependencies
-  * For node.js, use [`npm audit`](https://docs.npmjs.com/cli/audit) with [audit-filter](https://github.com/mozilla-services/audit-filter) to review and handle exceptions (see example in [speech-proxy](https://github.com/mozilla/speech-proxy/pull/77/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2))
-  * For Python, enable pyup security updates:
-    * Add a pyup config to your repo (example config: https://github.com/mozilla-services/antenna/blob/master/.pyup.yml)
-    * Enable branch protection for master and other development branches. Make sure the approved-mozilla-pyup-configuration team *CANNOT* push to those branches.
-    * From the "add a team" dropdown for your repo /settings page
-      * Add the "Approved Mozilla PyUp Configuration" team for your github org (e.g. for [mozilla](https://github.com/orgs/mozilla/teams/approved-mozilla-pyup-configuration) and [mozilla-services](https://github.com/orgs/mozilla-services/teams/approved-mozilla-pyup-configuration))
-      * Grant it write permission so it can make pull requests
-    * notify secops@mozilla.com to enable the integration in pyup
+  * [ ] [Enable automated security fix PRs on github](https://help.github.com/en/articles/configuring-automated-security-fixes#managing-automated-security-fixes-for-your-repository)
+  * For node.js, consider using [`npm audit`](https://docs.npmjs.com/cli/audit) with [audit-filter](https://github.com/mozilla-services/audit-filter) to review and handle exceptions (see example in [speech-proxy](https://github.com/mozilla/speech-proxy/pull/77/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2))
```

Comment:
Would it be obvious to a nodite whether `npm audit` is a replacement for GitHub's auto fix? Or should `npm audit` be in-addition-to?

As a non-nodeite, it isn't clear to me. Indentation may help clarify (both bullets are at the same level now -- only the `npm audit` one doesn't have a checkbox).

Reply:
oh yeah good call! It's an "in addition to", so should we drop that line or change it to:

> * [ ] For node.js projects, also run `npm audit` in CI with `audit filter` to filter exceptions (see example ...)

?

Reply:
If it provides additional value, we should keep it.

I assume, if it's an exception, they'd have to manually back out the auto-merged GitHub/Dependabot fix? Or, if not auto-merging, then dismiss the sec alert manually.

I.e. where do we draw the line between "you need X" and "here's how to use X in your workflow"?

Not being familiar with node, I'll defer to your judgement on that.

Reply:
We haven't defined the vuln alert workflow, but we used to recommend running `nsp audit` in CI, which changed to `npm audit` then `npm audit` with `audit-filter`  to avoid `|| true` and ignoring all npm audit vuln. alerts.

I don't think that recommendation is providing additional value commensurate with the confusion it causes, so I'm dropping it in this PR.

Whenever I get around to building a tool for this I'll put it in an "additional info / specific problems" section.